src/AutoPilotPlugins/PX4/ActuatorComponent.qml

```diff
@@ -257,24 +257,61 @@ SetupPage {
                                             }
                                         }
                                     }
+                                    WheelHandler {
+                                        onWheel: (event) => { // Capture the event object
+                                            let minimum = sliderRepeater.itemAt(0).channel.min;
```

Comment:
I don't like this piece of code since it assumes there is at least 1 actuator, and that all actuators have the same minimum and maximum. I am open to suggestions on how this could be improved

Reply:
I think that all having the same minimum and maximum is always the same now that I think about it since the Mavlink message sent to do the actuator test only allows a value from 0 to 1.